pyccel/codegen/printing/ccode.py

```diff
@@ -821,6 +821,10 @@ def _print_Module(self, expr):
         self.exit_scope()
         return code
 
+    def _print_PythonList(self, expr):
+        vec_declare = 'vec_' + expr.dtype._name
```

Comment:
Beware I don't think you have the right function name here

Reply:
It depends on what exactly you're trying to print, but it looks like it should be: `_print_HomogeneousListType`

But I don't know if `_print_Declare` uses `_print` to print the types anyway

Reply:
The class `PythonList` represents an object such as `[1,2,3]` not a type